app/controllers/bets_controller.rb

```diff
@@ -18,24 +18,32 @@ def edit
     @bet = find_bet
   end
 
-  # rubocop:disable Rails/ActiveRecordAliases
   def update
     @bet = find_bet
-    saver = SearchApiSaver.new(@bet)
     attrs =
```

Comment:
Do you think it might be worth extracting `attrs` and `notice` to small methods of their own?  It might clean this up a little.
You can then put `saver` closer to the the logic manipulating `@bet` which makes the flow easier to follow IMO.

Reply:
👍 good idea thanks